WordPress/Classes/Extensions/URL+IncrementalFilename.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+extension URL {
+
+    /// Returns a URL with an incremental file name, if a file already exists at the given URL.
+    ///
+    /// Previously seen in MediaService.m within urlForMediaWithFilename:andExtension:
+    ///
+    func incrementalFilename() -> URL {
```

Comment:
I went back and forth on this one many times. I settled on "incremental" as it seemed to match what's happening a bit more, where-as "incremented" seemed to imply that you could give it _any_ filename and it will increment it.

Since the method is a bit more on the _dumb_ side, it will find an incremented filename, eventually, hence "incremental". Could honestly go either way though.